src/cli/clusters_get.rs

```diff
@@ -75,13 +81,20 @@ fn clusters_get(
     } else {
         guard.active_capella_org()
     }?;
+
+    let project = if let Some(proj) = call.get_flag(engine_state, stack, "project")? {
+        proj
+    } else {
+        guard.active_project()?
+    };
```

Comment:
Matter of taste but I think this could also be written

```suggestion
    call.get_flag(engine_state, stack, "project")?.map_or_else(|| guard.active_project()?, Ok)?; 
```

I think I prefer yours :P, but it is [in the docs as an example](https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.map_or_else)

Reply:
I can't argue with the docs, so have done

Reply:
I asked my favorite rustacean and he said he prefers map_or_else when returning only one thing. And prefer the if let when there are possible side effects.

Reply:
Sounds like a good rule of thumb